components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -345,6 +345,17 @@ func (backend *ESClient) setYesterdayLatestToFalse(ctx context.Context, nodeId s
 	oneDayAgo := time.Now().Add(-24 * time.Hour)
 	indexOneDayAgo := mapping.IndexTimeseriesFmt(oneDayAgo)
 
+	time90daysAgo := time.Now().Add(-24 * time.Hour * 90)
+
+	// Making a filter query to get all the indices from today to 90 days back
+	filters := map[string][]string{"start_time": {time90daysAgo.Format(time.RFC3339)}, "end_time": {oneDayAgo.Format(time.RFC3339)}}
+
+	// Getting all the indices
+	esIndexs, err := relaxting.GetEsIndex(filters, true)
```

Comment:
Mark it as false. It will only give you summary indexes @dave-augustus

Reply:
Do we need to consider both report and summary indexes?

Reply:
We need to consider only reporting index